src/styles/index.scss

```diff
@@ -4,0 +1,7 @@
+@import '/src/styles/blocks/variables';
+@import '/src/styles/blocks/page';
+@import '/src/styles/blocks/card';
+@import '/src/styles/blocks/image';
+@import '/src/styles/blocks/cash';
+@import '/src/styles/blocks/rating';
```

Comment:
Ensure that the import path for the rating styles is correct and relative to the SCSS file location. The current path '/src/styles/blocks/rating' should be './blocks/rating' to align with the project structure.